CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v2.0.0-rc.1 / 2021-03-26
+
+* [CHANGE] Rename --labels-metric-allow-list to --metric-labels-allowlist #1424
+* [CHANGE] Remove deprecated Kubernetes APIs #1423
+* [CHANGE] Remove vendor folder #1419
+
```

Comment:
I would include the https://github.com/kubernetes/kube-state-metrics/commit/3ab8b19c29a0008e93171a11842cfe8a2c558686 as some of our users use kube-state-metrics as library, this should be helpful to them.

Reply:
Thanks, I added a change log.